ACPs/125-basefee-reduction/README.md

```diff
@@ -0,0 +1,42 @@
+```text
+ACP: 125
+Title: Reduce C-Chain base fee to 1 Gwei.
+Author(s): Stephen Buttolph <https://github.com/StephenButtolph>, Darioush Jalali <https://github.com/darioush>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+This ACP proposes reducing the base fee on the Avalanche C-Chain from 25 Gwei to 1 Gwei.
+
+## Motivation
+
+With dynamic fees, the gas price is supposed to be a result of a continuous auction such that the consumed gas per second converges to the target gas usage per second.
+When dynamic fees were first introduced safeguards were introduced to ensure that the mechanism worked as intended, such as a relatively high minimum gas price and a maximum gas price.
+The maximum gas price has since been entirely removed. The minimum gas price has been reduced significantly. However, it is still too high, and is therefore reducing usage of the network.
+
+## Specification
+
+The dynamic fee calculation currently must enforce a minimum base fee of 25 Gwei.
+This ACP proposes reducing the minimum base fee to 1 Gwei upon the next network upgrade activation.
+
+## Backwards Compatibility
+
+This ACP modifies the consensus rules for the C-Chain, therefore it requires a network upgrade.
+
+## Reference Implementation
+
+A draft implementation of this ACP for the coreth VM can be found [here](https://github.com/ava-labs/coreth/pull/604/files).
+
+## Security Considerations
+
+N/A
```

Comment:
Not sure how pedantic we want to be here.

This will increase state bloat (as it can make gas cheaper). But not past the parameterized bounds.

Reply:
I'd include that the dynamic fee algorithm responded appropriately during the inscription era which gives reasonable confidence that this min fee crutch is no longer necessary

Reply:
added, not sure if others prefer a different way to refer to the "inscription era"